core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -254,11 +254,65 @@ void validateAndApplyFeeAndQuote(
                 "Quote asset is [%s] different from the transaction sending asset [%s]",
                 quote.getSellAmount(), txn.getAmountIn()));
       }
+    }
+  }
+
+  void queryFee() throws AnchorException {
+    Sep31PostTransactionRequest request = Context.get().getRequest();
+    JwtToken token = Context.get().getJwtToken();
+    org.stellar.platform.apis.shared.Amount fee =
```

Comment:
nit: can we remove the full qualifier `org.stellar.platform.apis.shared.` and use `Amount` directly?